### ISSUE TITLE
Make message page keyboard-navigateable

### ISSUE THREAD

**Issue by robby1066** [robby1066/keepposted-help#1]
# Problem
Keep Posted messages should be accessible to people who don't have the use of a mouse. This could be people with limited motion or limited vision (who may still want to listen to the content of a keep posted message even if they can't see it.

# Solution
Adding proper tabindexes and making the javascript events respond to keydown actions would be a good start.

# Affects
The message player on both desktop and mobile
![image](https://user-images.githubusercontent.com/342510/100921522-4d3b2480-34a2-11eb-9ea6-da1b723245f0.png)

**Comment by robby1066** [comment]
This is related to #28

**Comment by robby1066** [comment]
This is related to #30

**Comment by robby1066** [comment]
Spent some time on this yesterday and today and have it mostly in a place I feel good about it. I want to test it out a fair bit before calling it good, though.

**Comment by robby1066** [comment]
Example of keyboard navigation on a message. Illustrating playing a message, toggling fast / normal playback, using the table of contents, interacting with feedback form, replaying message.


https://user-images.githubusercontent.com/342510/107127669-47d0d080-687d-11eb-945f-fa5521efd777.mp4

**Comment by robby1066** [comment]
Deployed on Feb 7, 2021
